Patches/Vanilla Ideology Expanded - Dryads/ThingDefs_Items/Items_Resource_Stuff.xml

```diff
@@ -27,31 +27,6 @@
 					</value>
 				</li>
 
-				<li Class="PatchOperationReplace">
-					<xpath>Defs/ThingDef[defName="VDE_Gaubric"]/stuffProps/statFactors/MaxHitPoints</xpath>
-					<value>
-						<MaxHitPoints>1</MaxHitPoints>
-					</value>
-				</li>
-
-				<li Class="PatchOperationConditional">
-					<xpath>Defs/ThingDef[defName="VDE_Gaubric"]/stuffProps/categories</xpath>
-					<nomatch Class="PatchOperationAdd">
-						<xpath>Defs/ThingDef[defName="VDE_Gaubric"]/stuffProps</xpath>
-						<value>
-							<categories>
-							  <li>SoftArmor</li>

```

Comment:
Why aren't we using it as soft armor anymore? Are we falling back to the description of it as being essentially leather?

Reply:
yea it's been nerfed down to a normal leather in the original mod